Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -199,5 +206,10 @@ public void updatePeriodicData() {
         
         leftDriveEncoderTicksProp.set(leftDrive.getPosition());
         rightDriveEncoderTicksProp.set(rightDrive.getPosition());
+        
+        Point leftPrimaryPoint = leftDrive.getTelemetryPoint(this.getClass().getSimpleName(), "left", true);
```

Comment:
This should only be done if the telemetry system is actually in use (here and everywhere else).